src/celeritas/ext/VecgeomTrackView.hh

```diff
@@ -230,6 +231,16 @@ CELER_FUNCTION VolumeId VecgeomTrackView::volume_id() const
     return VolumeId{this->volume().id()};
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Get the physical volume ID in the current cell.
+ */
+CELER_FUNCTION VolumeId VecgeomTrackView::volume_physid() const
```

Comment:
Since the result is not a "volume ID" (it can't be used as an input to anything that takes a `VolumeId`) just return it as an int or whatever the native type.

Reply:
Done.